### ISSUE TITLE
Error deploying sample clone

### ISSUE THREAD

**Issue by ZacharyKhan** [ParsePlatform/parse-server-example#112]
So I've been getting the error below while attempting to upload a .ZIP to ElasticBeanstalk. I downloaded the example Parse server and tried to deploy it without touching anything and still get this error from the EBS events log?

`The configuration file __MACOSX/parse-server-example-master/.ebextensions/._app.config in application version parse-server-base contains invalid YAML or JSON. YAML exception: unacceptable character '' (0x0) special characters are not allowed in "<reader>", position 0, JSON exception: Unexpected character () at position 0.. Update the configuration file.`

<img width="1031" alt="screen shot 2016-03-18 at 12 08 00 pm" src="https://cloud.githubusercontent.com/assets/14354004/13889120/19e2bf72-ed02-11e5-9697-dfe57aafbde1.png">

**Comment by gfosco** [comment]
What do you see in `.ebextensions/`?  The file looks okay to me in the repo..

**Comment by ZacharyKhan** [comment]
<img width="709" alt="screen shot 2016-03-18 at 12 30 42 pm" src="https://cloud.githubusercontent.com/assets/14354004/13889722/48904ad0-ed05-11e5-97ea-74a5c9fbe0a2.png">

I tried to deploy via EB CLI and it just timed out trying to execute.

**Comment by ZacharyKhan** [comment]
Not sure what solved the problem, I cloned the original server-file and re-filled in the default data to contact to my database/server and it seems to be going well.

**Comment by michalumni** [comment]
I have this issue as well...

**Comment by kesongxie** [comment]
I had the same issue. The reason is that when you download the currently deployed version from dashboard, modify the and zip it again, Apple will create some hidden file like __MACOSX along with the zip file. In order to resolve this, use the this command to zip the folder
`zip -r -X archive_name.zip folder_to_compress
`
Then upload this zip file to the dashboard again

**Comment by ctocph** [comment]
Thanks. For me the best option was to still zip it manually by right-clicking (because I got some other weird issues otherwise, like I couldn't keep the folder name separate from the unzipped folder name etc). Anyway. What I did to fix it all in the end was simply:
1. Zip it normally
2. Terminal `zip -d /Users/UserName/Path/To/File/app.zip __MACOSX/\*`
3. To make sure I run `zipinfo /Users/UserName/Path/To/File/app.zip`
4. Now AWS Beanstalk was happy.

Hope it helps.
